Update contao-manager.de.md

Ernestopheles opened the pull request:
Es fehlte bisher der Hinweis, dass man vor der Installation eine DB anlegen muss.

aschempp commented:
Uhm, the title is misleading? And I think these have been removed for security reasons?

lionel-m commented:
[Apparently not](https://github.com/contao/core/blob/split/core/contao/library/Contao/Controller.php#L1770-L1771). But these two flags [were removed for security reasons](https://github.com/contao/core/blob/1f7448f356cbef9299d7285b2b3068723e620eb1/system/docs/CHANGELOG.md#fixed-28) `base64_encode` and `base64_decode`. I will create a PR for that.

lionel-m commented:
> Uhm, the title is misleading?

I wrote ["from" instead of "for"](https://github.com/contao/docs/pull/211) in the last PR so "urlencode" and "rawurlencode" were removed in versions 3.3 and 3.4. That's why I created this PR

aschempp commented:
I think this can be closed, right?

lionel-m commented:
No because  "urlencode" and "rawurlencode" are present in the English documentation so they must also appear in French for versions 3.3 and 3.4.

aschempp commented:
But check the PR changes, it adds base64_encode and decode ;-)

lionel-m commented:
I had not seen this error !!! Normally this PR was planned for  "urlencode" and "rawurlencode".
Sorry... I'll recreate a PR.

Thank you for the check !

Ernestopheles committed:
Update contao-manager.de.md

Es fehlte bisher der Hinweis, dass man vor der Installation eine DB anlegen muss.

netzarbeiter reviewed (commented):
Das wird beim Installtool erklärt.
https://docs.contao.org/manual/de/installation/contao-installtool/
